api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -561,6 +561,11 @@ type AzureManagedMachinePoolSpec struct {
 	// Immutable.
 	// +optional
 	EnableFIPS *bool `json:"enableFIPS,omitempty"`
+
+	// EnableEncryptionAtHost indicates whether host encryption is enabled on the node pool.
+	// Immutable.
```

Comment:
Could we also include a link to the AKS docs for this feature like this?
```suggestion
	// Immutable.
	// See also [AKS doc].
	//
	// [AKS doc]: https://learn.microsoft.com/en-us/azure/aks/enable-host-encryption
```